src/ontology/mondo.Makefile

```diff
@@ -640,6 +641,21 @@ merge_obsolete_template: tmp/heal_hierarchy.ru $(MERGE_TEMPLATE) tmp/remove_clas
 	remove -T tmp/remove_classes.txt --preserve-structure false \
 	template --merge-before --template $(MERGE_TEMPLATE) convert -f obo -o $(SRC)
 
+ALL_SOURCES_DEPRECATED_PATTERNS = $(patsubst %, tmp/deprecated-%-mappings.robot.tsv, $(SOURCE_IDS))
+
+tmp/deprecated-%-mappings.robot.tsv:
+	wget "https://raw.githubusercontent.com/monarch-initiative/mondo-ingest/main/src/ontology/reports/$*_mapped_deprecated_terms.robot.template.tsv" -O $@
+
+update_deprecated_mappings: $(ALL_SOURCES_DEPRECATED_PATTERNS)
+	$(ROBOT) template --prefix "CHR: http://purl.obolibrary.org/obo/CHR_" --prefix "sssom: https://w3id.org/sssom/" --merge-before --input $(SRC) \
+ 		$(patsubst %, --template %, $^) \
+ 		convert -f obo -o $(SRC)
+	$(MAKE) NORM && mv NORM $(SRC)
+
+deprecated_annotation_merging:
+	sed -i 's/source="MONDO:equivalentObsolete",\ source="MONDO:obsoleteEquivalentObsolete"/source="MONDO:obsoleteEquivalentObsolete"/g' mondo-edit.obo || true
+	sed -i 's/source="MONDO:equivalentObsolete",\ source="MONDO:equivalentTo"/source="MONDO:equivalentObsolete"/g' mondo-edit.obo || true
+	#(.*)source="MONDO:equivalentObsolete"(.*)source="MONDO:obsoleteEquivalentObsolete"(.*) // 
```

Comment:
@joeflack4, can you turn this comment into an `sed` command? I provide some pseudocode below. This does not work, and also causes a bug if the second `source=` block is the last in the OBO xref (e.g. `xref: OMIM:601251 {source="MONDO:equivalentObsolete", source="GARD:0010117", source="MONDO:obsoleteEquivalentObsolete"}`). We do _not_ want the `,` before the `}` symbol. 

```suggestion
	sed -i 's/(.*)source="MONDO:equivalentObsolete"(.*)source="MONDO:obsoleteEquivalentObsolete"(.*)/$1source="MONDO:obsoleteEquivalentObsolete"$2$3/g' mondo-edit.obo || true
```

Reply:
Closed in favour of #6345